### PR TITLE
[accumulo1.6, scripts] deprecate Accumulo 1.6 client, since Accumulo 1.6 is EOM.

### PR DESCRIPTION
**Pull request by busbey**
According to http://accumulo.apache.org/release/accumulo-1.6.6/

### CODE REVIEW

**Comment by busbey** [comment]
@joshelser think you could take a quick look at this?

File: bin/ycsb.sh [hunk 1]
```diff
@@ -142,6 +142,14 @@ be removed in a future YCSB release."
   BINDING_DIR="accumulo1.6"
```
Comment (ctubbsii) [inline bin/ycsb.sh]: Should also change the default binding for `accumulo` to a non-1.6 version.

**Comment by busbey** [comment]
the binding name `accumulo` has already been deprecated and is planned for removal at the same time as the `accumulo1.6` binding

**Comment by busbey** [comment]
Went back to look and #1034 (which deprecated the bare `accumulo` name) hasn't made it into a release yet. So we could keep it if we want to. But I've been trying to push any module with version-specific clients to always specify the version so we don't have to do breaking changes when we change where a bare name points. thoughts?

**Comment by ctubbsii** [comment]
I guess the only risk with `accumulo` pointing to `accumulo1.6` is that the user gets 2 warnings, which is probably fine.

**Comment by busbey** [comment]
thanks for the review @ctubbsii!

**Pull request merged**